modules/oe_theme_content_organisation/src/Plugin/ExtraField/Display/TeaserDetailsExtraField.php

```diff
@@ -85,32 +79,38 @@ public function viewElements(ContentEntityInterface $entity) {
       ],
     ];
     $cache = CacheableMetadata::createFromRenderArray($build);
+    $items = [];
 
-    $contact_access = $contact->access('view', NULL, TRUE);
-    $cache->addCacheableDependency($contact);
-    $cache->addCacheableDependency($contact_access);
+    /** @var \Drupal\oe_content_entity_contact\Entity\ContactInterface $contact */
+    foreach ($entity->get('oe_organisation_contact')->referencedEntities() as $contact) {
+      if (!$contact) {
+        continue;
+      }
+      $contact_access = $contact->access('view', NULL, TRUE);
+      $cache->addCacheableDependency($contact);
+      $cache->addCacheableDependency($contact_access);
 
-    if (!$contact_access->isAllowed()) {
-      $cache->applyTo($build);
-      return $build;
-    }
+      if (!$contact_access->isAllowed()) {
+        $cache->applyTo($build);
+        return $build;
```

Comment:
So if the first contact is not accessible, we return and don't continue the loop?